### PR TITLE
Madhura_Assignment_8_Done

### PR DESCRIPTION
**Pull request by MadhuraMJun20**
Assignment 8 done

### CODE REVIEW

File: src/madhuraMJun20/BankingSystem.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package madhuraMJun20;
+
+public class BankingSystem {
+	
+	int counterD=0;
+	int counterC=0;
+	int counterP=0;
+	static int countD=0;
+	static int countC=0;
+	static int countP=0;
```
Comment (KrishnaKTechnocredits) [inline src/madhuraMJun20/BankingSystem.java]: Remove the initialization of all instance variables.

File: src/madhuraMJun20/BankingSystem.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	
+	void setData(int credit, int debit, int print) {
+
```
Comment (KrishnaKTechnocredits) [inline src/madhuraMJun20/BankingSystem.java]: Why are you not set here anything? Then what's the purpose of creating this method.

File: src/madhuraMJun20/BankingSystem.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		
+	}
+	void debit() {
+		countD++;
+		counterD++;
+	}
+	
+	void credit() {
+		countC++;
+		counterC++;
```
Comment (KrishnaKTechnocredits) [inline src/madhuraMJun20/BankingSystem.java:25]: You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.

File: src/madhuraMJun20/BankingSystem.java (hunk 1 above)
Comment (MadhuraMJun20) [reply src/madhuraMJun20/BankingSystem.java]: Done.

File: src/madhuraMJun20/BankingSystem.java (hunk 2 above)
Comment (MadhuraMJun20) [reply src/madhuraMJun20/BankingSystem.java]: Corrected that.

File: src/madhuraMJun20/BankingSystem.java (hunk 3 above)
Comment (MadhuraMJun20) [reply src/madhuraMJun20/BankingSystem.java:25]: Done.

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**